src/Command/LazyLoadingCommand.php

```diff
@@ -50,12 +50,22 @@ public function __construct(string $name, string $commandClass, ContainerInterfa
         $this->setHelp($command->getHelp());
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output) : int
+    /**
+     * @return int
+     */
+    public function run(InputInterface $input, OutputInterface $output)
```

Comment:
We can add RTH here. It might be added in the next version of Symfony, so adding it now can safe us :)